external/cblas/include/cblas.h

```diff
@@ -562,6 +562,16 @@ void cblas_zher2k(const enum CBLAS_ORDER Order, const enum CBLAS_UPLO Uplo,
 
 void cblas_xerbla(int p, const char *rout, const char *form, ...);
 
+// blas-like extensions
+void cblas_somatcopy(const enum CBLAS_ORDER CORDER,
```

Comment:
Thanks for your patch, @s-Nick.

Not all the BLAS implementations support these functions, luckily OpenBLAS does. However, CMake looks for the system BLAS library during configuration, which could not support them, e.g., in the case of BLAS reference implementation.

Reply:
Thank you for highlighting it.

As we discussed offline, I implemented an host version for these operators not to force dependencies on a specific BLAS implementation. Changes were commited in 49f48a5